wagtail-{{ cookiecutter.project_name_kebab }}/README.md

```diff
@@ -33,3 +33,30 @@
 
 - `pip install {{ cookiecutter.project_name_kebab }}`
 - ...
+
+## Contributing
+
+### Install
+
+To make changes to this project, first clone this repository:
+
+```sh
+git clone https://github.com/{{ cookiecutter.github_username }}/wagtail-{{ cookiecutter.project_name_kebab }}.git
```

Comment:
I've assumed we will go with the changes I've proposed in https://github.com/wagtail/cookiecutter-wagtail-package/pull/31 here. If we want to hard-code `wagtail`, I can change this

Reply:
I should have already resolved these. https://github.com/wagtail/cookiecutter-wagtail-package/pull/31 was merged, keeping `github_username` as a param and applying it consistently.